src/adapters/convex/products/extra-reward/convexExtraRewardAdapter.ts

```diff
@@ -328,7 +328,15 @@ export class ConvexExtraRewardAdapter
     const poolMetadata = (await this.buildMetadata())[protocolTokenAddress]
 
     if (!poolMetadata) {
-      logger.error({ protocolTokenAddress }, 'Protocol token pool not found')
+      logger.warn(
```

Comment:
`logger.error`

Reply:
The issue here is not every protocol token will have rewards available.

Reply:
So we were getting error thrown